src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -185,3 +186,232 @@ If you publish your template as *Unlisted*, you can test it under exactly the sa
+
+*Thumbnail* shows previews in both square and rectangular format.
+
+==== Admin
+
+If the admin context is enabled, selecting *Admin* displays the admin page.
+Events associated with the admin context are displayed in an events pane.
+See <<_viewing_events>>.
+
+==== About
+
+If the about context is enabled, clicking *About* displays the page that appears when the user clicks *Remix this Koji*.
+Events associated with the about context are displayed in an events pane.
+See <<_viewing_events>>.
+
+==== Sticker
+
+The *Sticker* context shows you how your template behaves when it's embedded inside another template.
+Events associated with the sticker context are displayed in an events pane.
+See <<_viewing_events>>.
+
+==== Dynamic Receipt
+
+If a dynamic receipt is enabled, a page is displayed for editing it.
+See <<withkoji-koji-iap-package#, in-app purchases>>.
+
+=== Viewing events
+
+The template generates events that are sent to the Koji platform, and the Koji platform generates events that are sent to the template.
+In most cases, selecting one of the contexts in the debugger displays an events pane which shows the events associated with that context.
+An events pane allows you to see in real time the events that are generated as you interact with the template in the selected context.
+Events generated by the template are listed on the right side of the events pane.
+Events generated by the Koji platform are listed on the left side.
+
+The info for each event includes:
+
+* The type of event – PLATFORM or TEMPLATE
+* The name of the event – For example: *Checked*
+* Properties set by the event (optional) – This may include descriptions of the properties.
+* New values for the properties (optional) – The new values for the properties set by the event.
+
+If you open a template and select the *Remix* context, here's an example of some events that you might see in the events pane:
+
+* TEMPLATE +
+Ready
+
+* PLATFORM +
+Changing remix state +
+VALUE +
+Remixing
+
+* TEMPLATE +
+Set value +
+PATH +
++++[general, title]+++ +
+NEW VALUE +
++++""+++
+
+If you click in an editable field on the remixing page, a *Clicked* event will appear in the event pane.
+For example:
+
+* TEMPLATE +
+Clicked +
+POSITION +
+(131, 583)
+
+If you then change the value of the field you clicked, a *Set Value* event will appear.
+For example:
+
+* TEMPLATE +
+Set value +
+PATH +
++++[general, goal]+++ +
+NEW VALUE +
+"21.00" +
+TYPE +
+Skip update
+
+=== Info and Features
+
+You can view and modify environment info for your template, as well as settings and data for implemented features.
+If you click on a feature that is not implemented, a message will be displayed saying that no data was found for the feature.
+
+The *Info and Features pane* includes the following panels.
+
+==== Environment info
+
+The *Environment info* panel displays your templates's *App Id*, *Frame URL*, and *VCC editor source*.
+You can click *Modify* to edit the frame URL.
```

Comment:
I actually don’t see a place to edit the Frame URL, so I don’t know if I’m missing something or if the UI has changed. We might need some tech info from Sean.

Reply:
Hmm... maybe it did change. Or maybe I was seeing things.